src/lbaf/Model/lbsObject.py

```diff
@@ -3,25 +3,49 @@
 from .lbsBlock import Block
 from .lbsObjectCommunicator import ObjectCommunicator
 
-
 class Object:
-    """A class representing an object with load and communicator"""
+    """A class representing an object with load and communicator
+
+    Constructor arguments:
+    :arg seq_id: small int identifier, commonly the task index, defaults to None
+    :arg packed_id: alt identifier (generated by vt as a bitpacked value), defaults to None
+    :arg r_id: rank ID, defaults to None
+    :arg load: the load, defaults to 0.0
```

Comment:
replace with `load: the computational time, also known as load, defaults to 0.0`

Reply:
Done